roles/KubevirtCommonTemplatesBundle/defaults/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 # defaults file for KubevirtCommonTemplatesBundle
-version: "{{ templates_version |default('v0.7.0') }}"
-
+version: "{{ templates_version |default('v0.6.2') }}"
```

Comment:
I think you can remove the default() part here.

Reply:
done